packages/editor-ui/src/components/ExecutionsList.vue

```diff
@@ -40,12 +40,7 @@
 				<thead>
 					<tr>
 						<th>
-							<el-checkbox
-								:indeterminate="isIndeterminate"
-								v-model="checkAll"
-								@change="handleCheckAllChange"
-								label=""
-							/>
+							<el-checkbox :value="checkAll" @change="handleCheckAllChange" label="" />
```

Comment:
this seems like an intentional feature we are losing here.. can you make sure to double check with Product and design first?